src/appBootstrapper.jsx

```diff
@@ -36,7 +36,7 @@ import getBrowserWindow from "shared/lib/getBrowserWindow";
 import {getConfigurationServiceApiUrl} from "shared/api/urls";
 import {AppStore} from "./AppStore";
 
-superagentCache(superagent);
+//superagentCache(superagent);
```

Comment:
Caching causes a bug in the search bar (if the load more option clicked for a term for the second time, the dropdown disappears), so I'm turning it off for now until I find a fix, so the heroku demo wouldn't be buggy.